archive_viewer/mixins/traces_table.py

```diff
@@ -89,6 +91,38 @@ def curve_delegates_init(self) -> None:
         delete_row_del = DeleteRowDelegate(self.ui.traces_tbl)
         self.ui.traces_tbl.setItemDelegateForColumn(delete_col, delete_row_del)
 
+    def dragEnterEvent(self, e):
+        """Handle something (like PV names) being dragged into the table"""
+        e.accept()
```

Comment:
It looks like the [drag and drop documentation](https://doc.qt.io/qtforpython-5/overviews/dnd.html#drag-and-drop) suggests using `acceptProposedAction()` instead of `accept()`.
From what I can tell, `accept()` should be used when you change the proposed action, docs on that [here](https://doc.qt.io/qtforpython-5/PySide2/QtGui/QDropEvent.html#PySide2.QtGui.PySide2.QtGui.QDropEvent:~:text=Once%20a%20replacement%20drop%20action%20has%20been%20set%2C%20call%20accept()).